### PR TITLE
Solves issue #9

### PR DESCRIPTION
**Pull request by y-ashaswini**
Le Magique Squares ;)

### CODE REVIEW

File: src/data/content.js
Comment (ken1000minus7) [inline src/data/content.js:1]: Remove the indentation changes in this file

File: src/pages/Activities.js
Comment (ken1000minus7) [inline src/pages/Activities.js:1]: Remove the indentation changes in this file

File: src/pages/Games.js
Comment (ken1000minus7) [inline src/pages/Games.js:1]: Remove the indentation changes in this file

File: src/data/content.js [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
-        urlTerm: "tic-tac-toe",
-        element: <TicTacToe />
-    }
-]
\ No newline at end of file
+  {
+    urlTerm: "magicsquares",
+    title: "Magic Squares",
+    description: "Magic Squares > contribution by y-ashaswini :)",
+    icon: "https://cdn-icons-png.flaticon.com/512/2076/2076261.png",
```
Comment (ken1000minus7) [inline src/data/content.js:20]: Use a different icon for this game

File: src/pages/Games.js [hunk 2]
```diff
@@ -1,20 +1,16 @@
-import "../styles/pages/Games.css"
+// import "../styles/pages/Games.css"
```
Comment (ken1000minus7) [inline src/pages/Games.js:1]: Any reason for commenting this?

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]
Game looks great, just a few changes required

File: src/pages/Games.js (hunk 2 above)
Comment (y-ashaswini) [reply src/pages/Games.js:1]: Yes, its empty...

Comment (y-ashaswini) [reply src/pages/Games.js:1]: It's counterpart, activities.css is empty too, and it isn't included in activities.js. so i removed it here too.

**Review by ken1000minus7 — APPROVED** [review]

**Pull request merged**